src/picongpu/include/plugins/output/GatherSlice.hpp

```diff
@@ -55,7 +62,9 @@ struct GatherSlice
      */
     bool init(bool isActive)
     {
-        /*free old communicator of init is called again*/
+        static int masterRankOffset = 0;
```

Comment:
A `static` method member is shared over all instances: http://stackoverflow.com/questions/6223355/static-variables-in-class-methods

I have also checked my changes and come to the same result like shown in the stackoverflow post.